assets/src/components/progressBar/style.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * Progress bar styles.
+ */
+
+.muriel-progress-bar {
+	margin: 16px 0;
+	position: relative;
+	z-index: 1;
+
+	.muriel-progress-bar__headings {
+		font-size: 16px;
+		margin-bottom: 1em;
+		color: $muriel-gray-600;
+		display: flex;
+		justify-content: space-between;
+		font-weight: 500;
+	}
+
+	.muriel-progress-bar__fraction {
+		margin-left: auto;
+	}
+
+	.muriel-progress-bar__bar {
+		height: 4px;
+		background-color: $muriel-hot-pink-500;
```

Comment:
I wonder if a red-ish color like this tracks as indicating errors rather than progress? Let's revisit with Sonja, but I'm wondering if it should be something more  like `$muriel-blue-400`